### ISSUE TITLE
[Accessibility][ContextMenu]: Disabled items should be focusable

### ISSUE THREAD

**Issue by jeschu1** [OfficeDev/office-ui-fabric-react#1156]
### Bug Report
Package version(s): 1.7.2
Browser and OS versions: All

#### Describe the issue:
Disabled menu items should be focusable so they can be read by screen readers

#### Actual behavior:
Disabled menu items are skipped 

#### Expected behavior:
Disabled menu items are focusable with an indicator that they can't be actioned

**Comment by battletoilet** [comment]
@betrue-final-final is this bug still accurate?

**Comment by betrue-final-final** [comment]
Yes, this is true. All of our disabled controls should be focusable.
